### PR TITLE
Update xwidgets patch to a newer one

### LINKED ISSUE #94: --with-xwidgets seems to be broken
```
18:27:28 ~/.emacs.d [develop]$ brew install emacs-plus --with-xwidgets --HEAD
==> Installing emacs-plus from d12frosted/emacs-plus
==> Installing dependencies for d12frosted/emacs-plus/emacs-plus: gnu-sed, texinfo, graphite2, harfbuzz, imagemagick@6
==> Installing d12frosted/emacs-plus/emacs-plus dependency: gnu-sed
==> Downloading https://homebrew.bintray.com/bottles/gnu-sed-4.5.high_sierra.bottle.tar.gz
######################################################################## 100.0%
==> Pouring gnu-sed-4.5.high_sierra.bottle.tar.gz
==> Caveats
The command has been installed with the prefix "g".
If you do not want the prefix, install using the "with-default-names" option.

If you need to use these commands with their normal names, you
can add a "gnubin" directory to your PATH from your bashrc like:
  PATH="/usr/local/opt/gnu-sed/libexec/gnubin:$PATH"

Additionally, you can access their man pages with normal names if you add
the "gnuman" directory to your MANPATH from your bashrc as well:
  MANPATH="/usr/local/opt/gnu-sed/libexec/gnuman:$MANPATH"
==> Summary
🍺  /usr/local/Cellar/gnu-sed/4.5: 12 files, 529KB
==> Installing d12frosted/emacs-plus/emacs-plus dependency: texinfo
==> Downloading https://homebrew.bintray.com/bottles/texinfo-6.5.high_sierra.bottle.tar.gz
######################################################################## 100.0%
==> Pouring texinfo-6.5.high_sierra.bottle.tar.gz
==> Caveats
This formula is keg-only, which means it was not symlinked into /usr/local,
because software that uses TeX, such as lilypond and octave, require a newer
version of these files.

If you need to have this software first in your PATH run:
  echo 'export PATH="/usr/local/opt/texinfo/bin:$PATH"' >> ~/.bash_profile

For compilers to find this software you may need to set:
    LDFLAGS:  -L/usr/local/opt/texinfo/lib

==> Summary
🍺  /usr/local/Cellar/texinfo/6.5: 408 files, 8.3MB
==> Installing d12frosted/emacs-plus/emacs-plus dependency: graphite2
==> Downloading https://homebrew.bintray.com/bottles/graphite2-1.3.12.high_sierra.bottle.tar.gz
######################################################################## 100.0%
==> Pouring graphite2-1.3.12.high_sierra.bottle.tar.gz
🍺  /usr/local/Cellar/graphite2/1.3.12: 18 files, 235.9KB
==> Installing d12frosted/emacs-plus/emacs-plus dependency: harfbuzz
==> Downloading https://homebrew.bintray.com/bottles/harfbuzz-1.8.8.high_sierra.bottle.tar.gz
######################################################################## 100.0%
==> Pouring harfbuzz-1.8.8.high_sierra.bottle.tar.gz
🍺  /usr/local/Cellar/harfbuzz/1.8.8: 151 files, 6.3MB
==> Installing d12frosted/emacs-plus/emacs-plus dependency: imagemagick@6
==> Downloading https://homebrew.bintray.com/bottles/imagemagick@6-6.9.10-10.high_sierra.bottle.tar.gz
######################################################################## 100.0%
==> Pouring imagemagick@6-6.9.10-10.high_sierra.bottle.tar.gz
==> Caveats
This formula is keg-only, which means it was not symlinked into /usr/local,
because this is an alternate version of another formula.

If you need to have this software first in your PATH run:
  echo 'export PATH="/usr/local/opt/imagemagick@6/bin:$PATH"' >> ~/.bash_profile

For compilers to find this software you may need to set:
    LDFLAGS:  -L/usr/local/opt/imagemagick@6/lib
    CPPFLAGS: -I/usr/local/opt/imagemagick@6/include
For pkg-config to find this software you may need to set:
    PKG_CONFIG_PATH: /usr/local/opt/imagemagick@6/lib/pkgconfig

==> Summary
🍺  /usr/local/Cellar/imagemagick@6/6.9.10-10: 1,471 files, 23MB
==> Installing d12frosted/emacs-plus/emacs-plus --HEAD --with-xwidgets
==> Cloning https://github.com/emacs-mirror/emacs.git
Cloning into '/Users/mayvenn/Library/Caches/Homebrew/emacs-plus--git'...
remote: Counting objects: 4062, done.
remote: Compressing objects: 100% (3825/3825), done.
remote: Total 4062 (delta 304), reused 1073 (delta 140), pack-reused 0
Receiving objects: 100% (4062/4062), 39.62 MiB | 6.38 MiB/s, done.
Resolving deltas: 100% (304/304), done.
Checking out files: 100% (3892/3892), done.
==> Checking out branch master
Already on 'master'
Your branch is up to date with 'origin/master'.
==> Downloading https://gist.githubusercontent.com/aatxe/260261daf70865fbf1749095de9172c5/raw/214b50c62450be1cbee9f11cecba846dd66c7d06/patch-m
######################################################################## 100.0%
==> Downloading https://gist.githubusercontent.com/fuxialexander/0231e994fd27be6dd87db60339238813/raw/b30c2d3294835f41e2c8afa1e63571531a38f3cf
######################################################################## 100.0%
==> Patching
==> Applying patch-multicolor-font.diff
patching file src/macfont.m
Hunk #1 succeeded at 2345 (offset -28 lines).
==> Applying 0_all_webkit.patch
patching file configure.ac
Hunk #2 succeeded at 2710 (offset -31 lines).
patching file lisp/xwidget.el
patching file nextstep/templates/Info.plist.in
patching file src/Makefile.in
Hunk #1 succeeded at 417 (offset 3 lines).
patching file src/emacs.c
Hunk #1 succeeded at 1577 (offset 36 lines).
Hunk #2 succeeded at 1653 (offset 36 lines).
patching file src/nsterm.m
patching file src/nsxwidget.h
patching file src/nsxwidget.m
patching file src/xwidget.c
Hunk #25 FAILED at 990.
Hunk #31 succeeded at 1444 with fuzz 1.
1 out of 31 hunks FAILED -- saving rejects to file src/xwidget.c.rej
patching file src/xwidget.h
patch unexpectedly ends in middle of line
Error: Failure while executing; `patch -g 0 -f -p1 -i /private/tmp/emacs-plus--patch-20180819-43261-sswep/0_all_webkit.patch` exited with 1.
```

### PR DESCRIPTION
**Pull request by pcr910303**
Fixes #94 👍

### CODE REVIEW

**Comment by d12frosted** [comment]
Thank you very much. I will take a look at your PR very soon (tm).

**Comment by d12frosted** [comment]
Thank you very much. I will take a look at your PR very soon (tm).

P. S. do you mind to push you patch to the patches folder of this repository? It's safer to have all patches in one place, so if you for some reason remove your gist, others are not affected.

**Comment by pcr910303** [comment]
@d12frosted I added the patch to my branch and modified the formula accordingly; the formula won't work until you merge this PR as the address specifies your master branch :-)

File: Formula/emacs-plus.rb [hunk 1]
```diff
@@ -153,8 +153,8 @@ class EmacsPlus < Formula
       odie "--with-xwidgets is supported only on cocoa via xwidget webkit"
     end
     patch do
-      url "https://gist.githubusercontent.com/fuxialexander/0231e994fd27be6dd87db60339238813/raw/b30c2d3294835f41e2c8afa1e63571531a38f3cf/0_all_webkit.patch"
-      sha256 "f35b955aef31537d2ff163ec9bfcc2176dbcd0ea64f05440d98ec2988b82ce25"
+      url "https://raw.githubusercontent.com/d12frosted/homebrew-emacs-plus/master/patches/xwidgets_webkit_in_cocoa.patch"
+      sha256 "84bbff657c3823d5a671956a429a05a4f27accfc36b25d12ef14d6537df976ab"
```
Comment (d12frosted) [inline Formula/emacs-plus.rb]: I may be doing something wrong, but my hash is different. Actually the previous one:

```
λ curl https://raw.githubusercontent.com/d12frosted/homebrew-emacs-plus/581586bf896f266de8edb6cf9381a6285c690008/patches/xwidgets_webkit_in_cocoa.patch | sha256sum 
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 79413  100 79413    0     0   324k      0 --:--:-- --:--:-- --:--:--  325k
f35b955aef31537d2ff163ec9bfcc2176dbcd0ea64f05440d98ec2988b82ce25  -
```

**Comment by d12frosted** [comment]
OK, I am going to merge it and test 😸 hold tight!

**Comment by d12frosted** [comment]
If the hash is not right, I'll just update it.

**Comment by pcr910303** [comment]
@d12frosted Sorry for that; I'm not sure what mixed up;;

```
$ curl https://raw.githubusercontent.com/d12frosted/homebrew-emacs-plus/master/patches/xwidgets_webkit_in_cocoa.patch | shasum -a 256
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 79413  100 79413    0     0  63123      0  0:00:01  0:00:01 --:--:-- 63126
f35b955aef31537d2ff163ec9bfcc2176dbcd0ea64f05440d98ec2988b82ce25  -
```

**Comment by d12frosted** [comment]
NP, I will test it on my computer and will wait for brew to yell at me. Waiting for XCode to update :smile_cat:

**Comment by d12frosted** [comment]
Ok, the update will take some time. Since xwidget was broken anyways, it's not critical if the hash is wrong.

**Comment by pcr910303** [comment]
@d12frosted Oops :-( The patch is already failing at HEAD  :-|
I'll try fixing the patch tonight and make another PR...

**Comment by pcr910303** [comment]
@d12frosted Oh, it looks like I uploaded the wrong patch :-(
Sorry for that...

**Pull request merged**